wd_demo.au3

```diff
@@ -383,6 +383,18 @@ Func DemoCookies()
 	Local $sCookie = '{"cookie": {"name":"' & $sName & '","value":"' & $sValue & '"}}'
 	_WD_Cookies($sSession, 'add', $sCookie)
 	_WD_Cookies($sSession, 'Get', $sName)
+	
+	ConsoleWrite("- WD: Get all cookies:" & @CRLF)
+	$sAllCookies = _WD_Cookies($sSession, 'getall')
```

Comment:
I know there is no `Local`

take a look here:
https://github.com/Danp2/au3WebDriver/pull/194/commits/fd270abda7a5da32d5566469dd0b2eb1341dde49

it will be joined together.